vocabularies/Hierarchy.json

```diff
@@ -214,6 +226,13 @@
             "@Core.Description": "Identifiers of nodes to be shown",
             "@Core.OptionalParameter": {}
           },
+          {
+            "$Name": "ShowTargets",
+            "$Collection": true,
+            "@Core.Description": "Targets (for example, of end-user messages) to be shown",
+            "@Core.LongDescription": "Each target has the format of [`Messages/target`](#Messages) and identifies an entity\n            whose node in the current hierarchy is to be shown.",
```

Comment:
Are these message targets absolute or relative? Probably relative. Relative to the request?

Reply:
Yes, we rarely say something to that but for the implementation we will need to know. In https://ui5.sap.com/#/topic/fbe1cb5613cf4a40a841750bf813238e we describe this detail. Would it not be helpful to be a bit more specific here?